versions/1.13.2/inject/mcp/client/Start.java

```diff
@@ -14,7 +14,7 @@ public static void main(String[] args)
          * Working directory is used as gameDir if not provided
          */
         String assets = System.getenv().containsKey("assetDirectory") ? System.getenv("assetDirectory") : "assets";
-        Main.main(concat(new String[]{"--version", "mcp", "--accessToken", "0", "--assetsDir", assets, "--assetIndex", "1.13", "--userProperties", "{}"}, args));
+        Main.main(concat(new String[]{"--version", "mcp", "--accessToken", "0", "--assetsDir", assets, "--assetIndex", "1.13.1", "--userProperties", "{}"}, args));
```

Comment:
Should be 1.13.2

Reply:
1.13.1 is correct, there is no 1.13.2 asset index